lib/aggregate-translations.js

```diff
@@ -45,6 +45,18 @@ const defaults = (options = {}) => {
   return defaultConfig;
 };
 
+/**
+ * Aggregates translation files and initializes the appropriate internationalization loaders.
+ * @param {Object} options - The aggregation options.
+ * @param {string} options.baseDir - Directory to search from and to prepend to the output directory.
+ * @param {string|string[]} options.directories - Translation directory regex pattern(s) to glob, in addition to the default search patterns.
+ * @param {FileSystem} options.fileSystem - The filesystem to use to write the translation and loader files.
+ * @param {string[]} options.locales - The list of locale codes to aggregate.
+ * @param {string} options.outputDir - Output directory for the translation and loader files.
+ * @param {string|string[]} options.excludes - Translation directory regex pattern(s) to glob exclude from the search patterns.
+ * @param {string} options.format - The format of syntax to output the translations with. Possible values are 'es5' and 'es6'.
+ * @returns {array} - An array of supported locales.
+ */
 const aggregatedTranslations = (options) => {
```

Comment:
Side note, I thought it was strange I didn't find any references to this function searching Github, but I think it's because it's defined as a noun instead of a verb. `aggregat(ed)Translations`